.github/workflows/docker.yml

```diff
@@ -65,6 +65,7 @@ jobs:
     with:
       parent-image: ""
       image: docker-stacks-foundation
+      variant: default
```

Comment:
If you apply the suggestion above, then you won't need to specify `variant: default` here in most of the places.

Reply:
Added the default.